pkgs/tools/text/gtranslator/default.nix

```diff
@@ -0,0 +1,64 @@
+{ stdenv
+, fetchurl
+, meson
+, ninja
+, pkgconfig
+, itstool
+, python3
+, wrapGAppsHook
+, libxml2
+, libgda
+, gspell
+, glib
+, gtk3
+, gtksourceview4
+, gnome3
+}:
+
+stdenv.mkDerivation rec {
+  pname = "gtranslator";
+  version = "3.32.0";
+
+  src = fetchurl {
+    url = "mirror://gnome/sources/${pname}/${stdenv.lib.versions.majorMinor version}/${pname}-${version}.tar.xz";
+    sha256 = "1dqigah7x6h3afw5jcs3lw3h9y0acj72arcgxr89l6galvknr2xa";
+  };
+
+  nativeBuildInputs = [
+    meson
+    ninja
+    pkgconfig
+    itstool
+    python3
+    wrapGAppsHook
+  ];
+
+  buildInputs = [
```

Comment:
`gettextpo` https://gitlab.gnome.org/GNOME/gtranslator/blob/3.32.0/meson.build#L117 ?

Reply:
This is probably a relict libglade is not used since GTK 2, they also ask for `gtksourceview-2.0`.

Reply:
https://gitlab.gnome.org/GNOME/gtranslator/issues/71